pkgs/development/libraries/qt-6/functions/brokenMkspecsFilesDisable.sh

```diff
@@ -0,0 +1,53 @@
+# https://github.com/NixOS/nixpkgs/issues/214765
+
+# the functions qtPrepareTool and qtPrepareLibExecTool are defined in
+# qtbase-dev/mkspecs/features/qt_functions.prf
+
+brokenMkspecsFilesDisable() {
```

Comment:
should i rename the *.sh files to lowercase?

Reply:
the nixpkgs naming convention seems to be `broken-mkspecs-files-disable.sh`